src/components/_layouts/Navigation/Navigation.tsx

```diff
@@ -14,13 +14,41 @@ export const Navigation: React.FC<IProps> = ({ children }) => {
         <styled.Wrapper>
             <styled.Logo>Software Factory</styled.Logo>
             <styled.NavButtonsWrapper>
-                <Button
+                <styled.NavButton
                     color="primary"
                     onClick={() => history.push(paths.home)}
                 >
                     Home
-                </Button>
-                <IconButton size={32} name="ShoppingCart" />
+                </styled.NavButton>
```

Comment:
Please change to { NavButton } , don't use .styled

Reply:
Okay, I've just pushed corrected version of it :)